### ISSUE TITLE
User reported unable to buy and sell domains on RNS testnet within Dapp browser

### ISSUE THREAD

**Issue by chrisli30** [rsksmart/rwallet#545]
We need to go through the buy and the sell process to see what happens.

https://marketplace.testnet.rifos.org/

**Comment by imstar15** [comment]
rwallet
![image](https://user-images.githubusercontent.com/16951509/95961457-a50cc600-0e37-11eb-90b1-1c0486c78802.png)

imtoken
![image](https://user-images.githubusercontent.com/16951509/95961474-ab02a700-0e37-11eb-9e43-1b4d1bd8e6a0.png)

两个钱包打开网站后，点击右上角会出现一个弹框，但是没有办法选择。

如果在mac的chrome打开网站，则可以
![image](https://user-images.githubusercontent.com/16951509/95961708-f9b04100-0e37-11eb-931f-2ad3a20d8235.png)

这里需要看下网站源码是如何判断客户端类型弹出选项的？点击选项之后执行了什么操作？我们rwallet如何适配到它的流程上？

**Comment by julianmrodri** [comment]
Will check why this might be happening. Probably we are only considering Nifty and Metamask.

**Comment by chrisli30** [comment]
@julianmrodri Yeah, could you try using tRIF faucet within RWallet Dapp browser? It works well in Dapp browser. As long as the site works with Metamask it will work for RWallet. It seems that in RNS mobile view the "Connect a wallet" function specifically looks for "Metamask" plugin. 

Screenshot from Chrome developer mode iPhone X view. In Chrome browser the site can recognize Metamask plugin, but that's not a universal way to load wallets. The code should really just call `window.ethereum` to find any compatible wallet.
![image](https://user-images.githubusercontent.com/2616844/96932250-70ba8a80-1473-11eb-904a-1c9e13dba06c.png)
